config/install/tripal_layout.tripal_layout_default_form.trpcultexperiment_layout.yml

```
@@ -0,0 +1,153 @@
+        weight: 3
+        show_empty: false
+        open: true
+        children:
+          - "exp_featureofinterest"
+      - id: expdesign_field
+        type: details
+        label: "Field"
+        weight: 1
+        show_empty: false
+        open: false
+        children:
+          - "exp_field_locations"
+          - "exp_field_design"
+      - id: expdesign_ghouse
+        type: details
+        label: "Greenhouse"
+        weight: 2
+        show_empty: false
+        open: false
+        children:
+          - "exp_ghouse_locations"
+          - "exp_ghouse_growingmedia"
+          - "exp_ghouse_design"
+      - id: expdesign_gchamber
+        type: details
+        label: "Growth Chamber"
+        weight: 3
+        show_empty: false
+        open: false
+        children:
+          - "exp_gchamber_locations"
+          - "exp_gchamber_growingmedia"
+          - "exp_gchamber_design"
+      - id: expdesign_bchem
+        type: details
+        label: "Biochemical Assay"
+        weight: 4
+        show_empty: false
+        open: false
+        children:
+          - "exp_bchem_technique"
+          - "exp_bchem_assayprotocal"
+          - "exp_bchem_labfacility"
+          - "exp_bchem_date"
+      - id: experiment_design
+        type: details
+        label: "Experimental Design"
+        description: "Experimental design metadata is often specific to the type of experiment being done. <strong>Please expand the sections specific to your experiment and fill out the metadata within.</strong> For example, if you are using a field experiment to collect your samples but also doing a biochemical assay to evaluate these samples, then you will fill out the metadata in both the 'Field' and 'Biochemical Assay' sections below."
+        weight: 4
+        show_empty: false
+        open: true
+        children:
+          - "expdesign_field"
+          - "expdesign_ghouse"
+          - "expdesign_gchamber"
+          - "expdesign_bchem"
+      - id: experiment_people
+        type: details
+        label: "Researchers"
+        description: "The fields are used to ensure everyone involved in the experiment recieves credit for the work they did."
+        weight: 5
+        show_empty: false
+        open: true
+        children:
+          - "exp_datacollector"
+          - "exp_custodian"
+          - "exp_curator"
+      - id: experiment_funding
+        type: details
+        label: "Funding"
+        description: "The fields ensure that all funding organizations behind this research recieve credit for funding this work. You may need to talk your PI for these details."
+        weight: 7
+        show_empty: false
+        open: true
+        children:
+          - "exp_org"
+      - id: experiment_attribution
+        type: details
+        label: "Attribution"
+        description: "The fields are used to ensure everyone involved in the experiment recieves credit for the work they did."
```

This might be more obvious once I can test in the UI, but why does this share the same description as line 114?

```suggestion
        description: "The fields are used to ensure everyone involved in the experiment receives credit for the work they did."
```